### ISSUE TITLE
Question regarding e2e tests

### ISSUE THREAD

**Issue by rettal** [dev-mastery/comments-api#6]
Hi, thank you for a great video and for sharing the code. I have a question regarding the end to end tests that jest runs:

Does the below line depend on the response from a call to a third party service? I am just trying to learn and understand the code.

https://github.com/dev-mastery/comments-api/blob/0d119f7b295cfa482c65a005f0191456b1d6974b/__test__/e2e.spec.js#L36

Thanks again.

**Comment by arcdev1** [comment]
Yes. If Akismet or Content Moderator fail, the comment will not be published and the test will fail.
